hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/properties/NftPropertyTest.java

```diff
@@ -40,7 +41,7 @@ void gettersWork() {
 
         // expect:
         assertEquals(aEntity, NftProperty.OWNER.getter().apply(aSubject));
-        assertEquals(aMeta, NftProperty.METADATA.getter().apply(aSubject));
+        assertArrayEquals(aMeta, (byte[]) NftProperty.METADATA.getter().apply(aSubject));
```

Comment:
Tutaj problemem jest to, że metadata jest zwracane w postacii `Object`
```java
        @Override
        public Function<UniqueTokenAdapter, Object> getter() {
            return UniqueTokenAdapter::getMetadata;
        }
```

W innych miejscach widziałem że podobnie ten problem ogarniaja. Jak macie inne pomysły to dawać :D

Reply:
Test się wywalał bo porówanywał referencje do obiektów a nie wartości buforów